extension/README.md

```diff
@@ -162,12 +162,12 @@ extension updates are available.
 To specify a channel to track for an extension, add it to the `privateExtensions.channels` settings object as shown in the example below:
 ```
 "privateExtensions.channels" :{
-    "extensionOne": "insiders",     // Tracks the 'insiders' dist-tag
-    "extensionTwo": "beta",         // Tracks the 'beta' dist-tag
-    "extensionThree": "1.0.0"       // Pins the extension to version 1.0.0
+    "publisherOne.extensionOne": "insiders",     // Tracks the 'insiders' dist-tag
+    "publisherTwo.extensionOne": "beta",         // Tracks the 'beta' dist-tag
+    "publisherOne.extensionTwo": "1.0.0"       // Pins the extension to version 1.0.0
 }
 ```
-Note that the key is the extension's npm package ID and the value is the desired channel.
+Note that the key is the extension's npm package ID (`publisherName.packageID`) and the value is the desired channel.
```

Comment:
I recommend using "publisher" and "name" here to match the exact names of the fields in package.json.

See https://github.com/microsoft/vscode/blob/f9b6372f0a0d00438ef1ae15317f87efaac23356/src/vs/workbench/services/extensions/common/extensionsRegistry.ts#L342 for existing documentation in VS Code itself, which uses `"${publisher}.${name}"` and then gives an example. I copied that for https://github.com/joelspadin-garmin/vscode-private-extension-manager/blob/89e2ae81d1d33e73dd52996d9bd8e7e01f9691cb/extension/extensions.private.schema.json#L33 as well.